client/src/components/Accordions.js

```diff
@@ -89,10 +89,44 @@ const StatelessPullDownAccordion = ({
   isExpanded,
   children,
   onToggle,
+  showPin,
+  sticky_data, //sticky data only applies if show pin is true
+  closeAccordion,
 }) => (
   <div aria-label={title} className="pull-down-accordion">
-    <div className="pull-down-accordion-header" onClick={onToggle}>
-      <button aria-label={get_accordion_label(isExpanded)}>{title}</button>
+    <div className="pull-down-accordion-header" style={{ display: "flex" }}>
+      <button
+        aria-label={get_accordion_label(isExpanded)}
+        onClick={onToggle}
+        style={{ flexGrow: 1, textAlign: "center" }}
+      >
+        {title}
+      </button>
+      {showPin && (
+        <button
+          style={{ float: "right", marginLeft: "2rem" }}
+          onClick={() => {
+            sticky_data.pin_pressed();
+            sticky_data.can_pin && closeAccordion();
+          }}
+        >
+          {sticky_data.can_pin ? (
+            <IconPin
+              height="25px"
+              width="25px"
+              vertical_align="top"
+              alternate_color="false"
+            />
+          ) : (
+            <IconUnpin
+              height="25px"
+              width="25px"
+              vertical_align="top"
+              alternate_color="false"
+            />
+          )}
+        </button>
+      )}
```

Comment:
None of this belongs in `Accordions`, being pinned isn't an "accordion" concept. 

To be fair, it's not really a `KeyConcepts` concept either. The best approach would be to try an make a higher order component, something like `PinnedContent`, that wraps arbitrary children and gives them sticky position/pinning behaviour. I'm not going that far right now, but the pinning should at least be isolated to `KeyConcepts`.

Maybe the pin icon should float off to the left of the pinned content? Or render it within `KeyConcepts` and do some hacky negative margin stuff, or something, to position it back over the accordion (as long as this isn't a generic wrapper and is only used within `KeyConcepts` to stick an `Accordion`).